azure-resources/Compute/virtualMachineScaleSets/recommendations.yaml

```diff
@@ -182,7 +182,7 @@
   recommendationTypeId: null
   recommendationControl: Scalability
   recommendationImpact: High
-  recommendationResourceType: Microsoft.Compute/virtualMachines
+  recommendationResourceType: Microsoft.Compute/virtualMachineScaleSets
```

Comment:
With the `recommendationResourceType` corrected, the `aprlGuid` still points to a file in the `Microsoft.Compute/virtualMachines` folder. There is no `df0ff862-814d-45a3-95e4-4fad5a244ba6.kql` in the `Compute/virtualMachineScaleSets/kql` folder.

Reply:
Is in the new commit.